code/game/gamemodes/incursion/incursion.dm

```diff
@@ -67,12 +67,12 @@
 	return "Intel suggests that the Syndicate have recently had high level meetings discussing your station, and are disgruntled due to recent classified events. A large terrorist force may wish to take the station by force."
 
 //===please merge heretics so these can be made not terrible===
-/datum/game_mode/proc/update_incursion_icons_added(datum/mind/incursion_mind)
```

Comment:
Why are you doing this?

Reply:
So the proc can be called without the gamemode being incursion.

It's called from SSgame.mode or whatever, but if the gamemode is dynamic the proc doesn't exist on the mode

Reply:
Oh I see, the proc exists on all gamemodes.. not just incursion, lol

Reply:
Yeah, please no change